### PR TITLE
display talents in ability build

### PR DESCRIPTION
**Pull request by howardchung**

### CODE REVIEW

File: src/components/Visualizations/inflictorWithValue.jsx [hunk 1]
```diff
@@ -59,7 +59,9 @@ export default (inflictor, value, type) => {
     const ttId = uuid.v4();
 
     if (ability) {
-      image = `${API_HOST}/apps/dota2/images/abilities/${inflictor}_lg.png`;
+      image = inflictor.indexOf('bonus') !== -1
+        ? '/assets/images/stats.png'
```
Comment (blukai) [inline src/components/Visualizations/inflictorWithValue.jsx]: Need to change image

Comment (howardchung) [inline src/components/Visualizations/inflictorWithValue.jsx]: Are there specific images for each talent?

Comment (blukai) [inline src/components/Visualizations/inflictorWithValue.jsx]: I think I can make it in svg. I'do it. 🦄

**Comment by howardchung** [comment]
they should be displayed at the correct levels already.

**Comment by howardchung** [comment]
since current images are broken, pushing this as a hotfix.  Can accept a PR for improved images later.

**Pull request merged**